AUTHORS

```diff
@@ -1,3 +1,3 @@
 Nicholas H.Tollervey (ntoll@ntoll.org)
 Matt Wheeler (m@funkyhat.org)
-
+Edmond Locard (L0C4RD@gmail.com)
```

Comment:
Very cute. If, by chance, you really are named after the famous French forensics expert please link me to some sort of online presence I can use to verify who you are. Otherwise, pseudonyms are no more helpful than an anonymous gmail username, even for people who probably work in some sort of infosec capacity. :-)